src/api/resources/SSX/SSXDataCollectionProcessingCellsHistogram.ts

```diff
@@ -9,9 +9,12 @@ class SSXDataCollectionProcessingCellsHistogramEntity extends SSXDataCollectionP
   }
 }
 
-export const SSXDataCollectionProcessingCellsHistogramResource =
-  createAuthenticatedSingletonResource({
-    path: '/ssx/datacollection/processing/cells/histogram/:dummy',
+export const SSXDataCollectionProcessingCellsHistogramEndpoint =
```

Comment:
now `SSXDataCollectionProcessingCellsHistogramEndpoint` instead of `SSXDataCollectionProcessingCellsHistogramResource` for clarity